pkg/limayaml/limayaml.go

```diff
@@ -319,3 +320,18 @@ type CACertificates struct {
 	Files          []string `yaml:"files,omitempty" json:"files,omitempty" jsonschema:"nullable"`
 	Certs          []string `yaml:"certs,omitempty" json:"certs,omitempty" jsonschema:"nullable"`
 }
+
+// Engine is the name of the container engine, e.g. "docker", "containerd", that contains a list of sockets to monitor.
+type Engine struct {
+	Sockets []string `yaml:"sockets,omitempty" json:"sockets,omitempty"`
+}
+
+type Kubernetes struct {
+	Configs []string `yaml:"configs,omitempty" json:"configs,omitempty"` // Paths to Kubernetes config files, e.g. "/etc/rancher/k3s/k3s.yaml"
```

Comment:
If you want to keep the comment, then it should follow Godoc style:

```suggestion
	// Configs is a list of Kubernetes config files, e.g. "/etc/rancher/k3s/k3s.yaml"
	Configs []string `yaml:"configs,omitempty" json:"configs,omitempty"`
```